security-oauth2-quickstart/src/test/resources/.notes

```diff
@@ -0,0 +1,66 @@
+curl -X POST \
```

Comment:
What the goal of this file?

Reply:
These where my notes when developing this qucikstart. I thought that I may create documentation using these commands.

Reply:
I will delete these files when I am going to squash commits.